host/integrationbase.go

```diff
@@ -236,6 +239,38 @@ func (s *IntegrationBase) getHistory(namespace string, execution *commonpb.Workf
 	return events
 }
 
+func (s *IntegrationBase) getLastEvent(namespace string, execution *commonpb.WorkflowExecution) *historypb.HistoryEvent {
+	events := s.getHistory(namespace, execution)
+	s.Require().NotEmpty(events)
+	return events[len(events)-1]
+}
+
+func (s *IntegrationBase) decodePayloadsString(ps *commonpb.Payloads) (r string) {
```

Comment:
ps -> photoshop?

Reply:
about `(r string)` 

plz define the return value within the function body and return the result

Reply:
I think these are acceptable for a three-line helper function. Also, they were just moved from another file